src/blocks/Biometrics/index.js

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
@JakeDawkins since native is the primary use case, let's make this `index.web.js` and make the native one `index.js`.